src/test_driver/linux-cirque/OnOffClusterTest.sh

```diff
@@ -39,7 +39,7 @@ function build_chip_lighting() {
     source "$REPO_DIR/scripts/activate.sh" >/dev/null
     set -x
     cd "$chip_light_dir"
-    gn gen out/debug >/dev/null
+    gn gen out/debug --args='bypass_rendezvous=true' >/dev/null
```

Comment:
this looks like one of the few places RendezvousServer is being exercised by CI.  if it's turned off here, where is it being tested?

Reply:
The RendezvousServer has never been tested. In the cirque tests we always manually set the Thread network. Moreover the rendezvous code can only work on BLE before and after the PR so it cannot be tested in the simulation CI which now only supports IP network.

Reply:
Should we add a GH issue to track this? Or is this nothing that can be done (i.e. CI simulation simply cannot do BLE?)

if there is nothing to be done, some documentation would be nice, maybe where the flag is defined.

Reply:
@gjc13 perhaps instead of a compile-time flag, you can do a runtime check.  I.e. if the Thread network has already been set, there is no need for the Rendezvous server to be exercised this way?

Reply:
In fact we cannot since in the cirque test we still set the Thread network after the server process is launched.